xml/System.Diagnostics/ActivityLink.xml

```diff
@@ -167,7 +175,15 @@ Activity links can be used to represent batched operations where an activity was
         <summary>Indicates whether the current activity link is equal to another object.</summary>
         <returns>
           <see langword="true" /> if the current activity link is equal to <paramref name="obj" />; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+    ## Remarks
+
+    <xref:System.Diagnostics.ActivityLink> equality is currently undefined. The current behavior is two ActivityLinks with the same ActivityContext and no tags will return true for Equals. Prior to .NET 7 ActivityLinks sharing an ActivityTagsCollection instance and the same ActivityContext would return true for Equals. Starting from .NET 7, Equals will always return false when tags are used. This may change in the future. Exercise caution in calling this method.
```

Comment:
Same:
```suggestion
    <xref:System.Diagnostics.ActivityLink> equality is currently undefined. The current behavior is that two <xref:System.Diagnostics.ActivityLink> instances with the same <xref:System.Diagnostics.ActivityContext> and no tags will return `true` for <xref:System.Diagnostics.ActivityLink.Equals(System.Object)>. Prior to .NET 7, xref:System.Diagnostics.ActivityLink> instances sharing an <xref:System.Diagnostics.ActivityTagsCollection> instance and the same <xref:System.Diagnostics.ActivityContext> would return `true` for <xref:System.Diagnostics.ActivityLink.Equals(System.Object)>. Starting from .NET 7, <xref:System.Diagnostics.ActivityLink.Equals(System.Object)> will always return `false` when tags are used. This may change in the future. Exercise caution in calling this method.
```